src/utils/azure_storage.ts

```diff
@@ -161,17 +178,20 @@ export function getBlobAsText(
  * @param blobService     the Azure blob service
  * @param containerName   the name of the Azure blob storage container
  * @param blobName        blob file name
+ * @param defaultValue    if you want a default value insted of BlobNotFound error
  */
 export async function getBlobAsObject<A, O, I>(
   type: t.Type<A, O, I>,
   blobService: azureStorage.BlobService,
   containerName: string,
-  blobName: string
+  blobName: string,
+  defaultValue?: O
 ): Promise<Either<Error, A>> {
```

Comment:
what we could do is threat the not found error as a special case and return: `Promise<Either<Error, Option<A>>>` - not found errors will be mapped to `Right<None>` (we do that in other places too like when we query cosmos or redis)

Reply:
Implemented in https://github.com/teamdigitale/io-functions-commons/pull/22